akka-stream-tests/src/test/scala/akka/stream/io/FileSourceSpec.scala

```diff
@@ -59,6 +59,17 @@ class FileSourceSpec extends StreamSpec(UnboundedMailboxConfig) {
     f
   }
 
+  val directoryInsteadOfFile = Files.createTempDirectory(fs.getPath("/"), "directory-instead-of-file")
+
+  val testFifo = {
+    val d = Files.createTempDirectory(fs.getPath("/"), "file-source-spec")
+    val f = d.resolve("test-fifo")
+    // Java or NIO does not supply a way to create named pipes/FIFOs, so we use the POSIX command `mkfifo`.
+    Runtime.getRuntime.exec(s"mkfifo $f")
+    Files.newBufferedWriter(f, UTF_8).append(TestText).close()
+    f
+  }
```

Comment:
Let's just skip testing this, forking shell command processes from a test does feel like it makes for fragile tests and a high chance of leaking stuff over time etc.

The reason we use jimfs is to make the test work consistently and fast across machines.